pages/company/Company.tsx

```diff
@@ -0,0 +1,36 @@
+import type { NextPage } from "next";
+import Head from "next/head";
+import { Hero } from "pages/company/components/Hero";
+import { OurStory } from "pages/about/components/OurStory";
+import { Gallery } from "pages/about/components/Gallery";
+
+export const Company: NextPage = () => {
+	return (
+		<>
+			<Head>
+				<title>Company</title>
+				<meta
+					name="description"
+					content="The world's first walking buddy app of its kind. Get yourself a walking buddy anytime and anywhere in just a few clicks!"
+				/>
+				<meta
+					name="keywords"
+					content="home, walking, pal, buddy, walk, about"
+				/>
+			</Head>
+
+			<main
+				style={{
+					display: "flex",
+					flexDirection: "column",
+					justifyContent: "center",
+					alignItems: "center",
+				}}
+			>
+				<Hero />
+				{/* <Gallery />
+				<OurStory /> */}
```

Comment:
Uncomment these two

Reply:
> Uncomment these two

These are not ready and are copied from the about page just for reference

Reply:
Oh my bad. 

in that case, you will recieve such 2 lines when you pull from dev. 2 components (`<Data>`, `<Story>`)are already added by other 2 contributors.